spotbugs/etc/findbugs.xml

```diff
@@ -689,6 +691,7 @@
           <BugPattern abbrev="JUA" type="JUA_DONT_ASSERT_INSTANCEOF_IN_TESTS" category="BAD_PRACTICE" experimental="true" />
           <BugPattern abbrev="CN" type="OVERRIDING_METHODS_MUST_INVOKE_SUPER" category="CORRECTNESS" />
           <BugPattern abbrev="CNT" type="CNT_ROUGH_CONSTANT_VALUE" category="BAD_PRACTICE"/>
+          <BugPattern abbrev="MOE" type="MOE_METHOD_INVOKED_IN_OPTIONAL_ORELSE" category="BAD_PRACTICE"/>
```

Comment:
The build is failing due to:

> Task :spotbugs:spotbugsMain
>    java.lang.IllegalArgumentException: Error: missing bug code for keyMOE

Consider adding a `<BugCode abbrev="MOE"/>` element to this file.

Reply:
done

Reply:
Still facing the same problem, maybe you need to add a similar element to `spotbugs/etc/messages.xml`.

Reply:
done. The missing location was `messages.xml`